module.h

```diff
@@ -479,6 +480,55 @@ namespace ws::module {
 
 
 
+    struct Token {
+        std::string content;
+        std::uint32_t type;
```

Comment:
Can probably use unsigned char or uint8_t for type, I can't see us using more than 256 different types of tokens.

```suggestion
        std::uint8_t type;
```